template/package.yaml

```diff
@@ -0,0 +1,111 @@
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: Unlicense
+
+name:                templateDAO
+version:             0.0.0.0
+license:             MIT
+license-file:        LICENSE
+author:              Serokell, Tocqueville Group
+git:                 git@github.com:tqtezos/baseDAO.git
+copyright:           2020 Unlicense
+
+extra-source-files:
+- README.md
+
+description:         Please see the README on GitHub at <https://github.com/tqtezos/baseDAO/template#readme>
```

Comment:
Perhaps someone will put it in their own repo, so this comment will look wrong. I see three options:
1. Write something different not referring to our repo that will hold for any DAO.
2. Not include `description` at all.
3. Add a TODO indicating that the user of this template ought to do something with this line.

Something similar applies to some metadata fields above (e. g. `author`).

Reply:
I think I'll go with 3 and 1 (where possible).